datafusion/core/src/datasource/physical_plan/parquet/opener.rs

```diff
@@ -150,6 +155,8 @@ impl FileOpener for ParquetOpener {
 
             metadata_timer.stop();
 
+            // Then the reader is linked here.
+            // Rok, I also need your help linking here.
             let mut builder =
                 ParquetRecordBatchStreamBuilder::new_with_metadata(reader, metadata);
```

Comment:
@rok I think this is fine in that the metadata already has all the needed encryption keys for the ParquetRecordBatchStreamBuilder but a review from you to confirm would be good.